Powershell/XblConfig/ConnectDevAccount.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace XblConfig
+{
+    using System;
+    using System.Management.Automation;
+    using System.Net;
+    using System.Net.Http;
+    using System.Security;
+    using Microsoft.Xbox.Services.DevTools.Authentication;
+
+    /// <summary>
+    /// <para type="synopsis">Logs in a dev account.</para>
+    /// </summary>
+    [Cmdlet(VerbsCommunications.Connect, "DevAccount")]
+    public class ConnectDevAccount : PSCmdletBase
+    {
+        private enum AccountSourceOption
+        {
+            XDP = DevAccountSource.XboxDeveloperPortal,
+            WindowsDevCenter = DevAccountSource.WindowsDevCenter,
+        }
+
+        /// <summary>
+        /// <para type="description">The user name of the account.</para>
+        /// </summary>
+        [Parameter(Mandatory = true, HelpMessage = "The user name of the account.", Position = 0, ValueFromPipeline = true)]
+        public string UserName { get; set; }
+
+        /// <summary>
+        /// <para type="description">The account source where the developer account was registered. Can be either 'WindowsDevCenter' or 'XDP'.</para>
+        /// </summary>
+        [Parameter(Mandatory = true, HelpMessage = "The account source where the developer account was registered. Can be either 'WindowsDevCenter' or 'XDP'.", Position = 0, ValueFromPipeline = true)]
+        [ValidateSet("XDP", "WindowsDevCenter")]
+        public string AccountSource { get; set; }
+
+        /// <inheritdoc/>
+        protected override void BeginProcessing()
+        {
+            this.RequiresAuthorization = false;
+        }
+
+        /// <inheritdoc/>
+        protected override void Process()
+        {
+            try
+            {
+                if (!Enum.TryParse(this.AccountSource, out AccountSourceOption source))
```

Comment:
Is this necessary?  ValidateSet should reject anything other than these properties.  Though I suppose if someone is calling these classes manually it could do weird stuff.